spotdl/youtube_tools.py

```diff
@@ -189,9 +191,16 @@ def _best_match(self, videos):
             if not self.meta_tags:
                 # if the metadata could not be acquired, take the first result
                 # from Youtube because the proper song length is unknown
-                result = videos[0]
-                log.debug(
-                    "Since no metadata found on Spotify, going with the first result"
+                result = None if len(videos)==0 else videos[0]
+                
+                if(len(videos)==0):
```

Comment:
PEP 8 matters:
```suggestion
                if len(videos) == 0:
```
Or even
```suggestion
                if result is None:
```

Reply:
Sure thing, took a break after a long time java work :\.